### PR TITLE
Configuration react navigation 13

### LINKED ISSUE #13: Configuration React Navigation
**REQUIREMENTS**

In this feature we are going to create the basic configuration for the navigation like the stak navigation and tab navigations, stack navigation is for the login and signUP and the TAB navigations work when you get in the app.

### PR DESCRIPTION
**Pull request by DiegoSpielzeug**
# Description
We make the configuration of react navigation, and make the navigations
"Is Login" which we use tab navigation and for "LogOut" we use stack
navigation.
 
  
  ## Screenshots 
Screenshot_1621381399

### CODE REVIEW

File: DenunciaCiudadana/App.js [hunk 1]
```diff
@@ -19,13 +22,17 @@ export default function App() {
     return subscriber;
   });
 
-  if (initializing) {
-    return null;
-  }
+  // if (initializing) {
+  //   return null;
+  // }
```
Comment (Ilyeo) [inline DenunciaCiudadana/App.js]: if we do not need this snippet we could delete it, don’t you think?

File: DenunciaCiudadana/App.js [hunk 2]
```diff
@@ -19,13 +22,17 @@ export default function App() {
...
-    return <SignUp />;
-  }
 
-  return <Login user={user} />;
+  return(
+       isSignedIn 
+        ? 
+          <IsLogin user={user}/> 
+        : 
+          <LogOut/>
```
Comment (Ilyeo) [inline DenunciaCiudadana/App.js]: just to improve the readability I'll prefer this piece of code in one line.

```js
isSignedIn ? <IsLogin user={user} /> : <LogOut />
```

File: DenunciaCiudadana/src/navigation/IsLogin.js [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+  }
+
+const IsLogin = () => {
+    return (
+    <NavigationContainer>
+        <Tab.Navigator
+            screenOptions={({ route }) => ({
+              tabBarIcon: ({ focused, color, size }) => {
+                let iconName;
+                if (route.name === 'Home') {
```
Comment (Ilyeo) [inline DenunciaCiudadana/src/navigation/IsLogin.js]: here happen something similar, we could improve the readability using a `switch` statement, maybe in another iteration, we could improve the `switch` by a `hash`.

**Review by Ilyeo — COMMENTED** [review]
looks pretty well, i added minor comments about how to improve the readability of the code

File: DenunciaCiudadana/src/navigation/IsLogin.js (hunk 3 above)
Comment (DiegoSpielzeug) [reply DenunciaCiudadana/src/navigation/IsLogin.js]: Ready I make a helper thath help me to dont have many code in the same component

**Review by jkzasori — APPROVED** [review]
It's Okay for me.

**Review by Ilyeo — APPROVED** [review]
lgtm

**Pull request merged**